modules/distribution-zones/src/test/java/org/apache/ignite/internal/distributionzones/rebalance/DistributionZoneRebalanceEngineTest.java

```diff
@@ -443,6 +451,23 @@ void replicasTriggersAssignmentsChangingOnDefaultZone(
         }
     }
 
+    private static void completeTablesConfigs(TablesConfiguration tablesConfiguration) {
```

Comment:
Why is this method needed?

Reply:
The code that processes rebalance converts `TableView` to `CatalogTableDescriptor` and fails if there is no primary key or columns, so it is a temporary ceremony that will go away when we switch to the Catalog completely

Reply:
can it be provided as part of the initial configuration inside `@InjectConfiguration`?